HACKING.md

```diff
@@ -0,0 +1,70 @@
+# Hacking on Origin Aggregated Logging
+
+## Building on OpenShift
+Choose the project you want to hold your logging infrastructure. It can be
+any project.  The project/namespace ```logging``` is the default.
```

Comment:
I'm also not sure saying "logging" is the default is correct. The default is the project that you're currently in.